diesel/src/sqlite/types/date_and_time.rs

```diff
@@ -18,3 +18,85 @@ impl types::HasSqlType<types::Timestamp> for Sqlite {
         SqliteType::Text
     }
 }
+
+#[cfg(feature = "chrono")]
+mod chrono {
+    extern crate chrono;
+
+    use std::error::Error;
+    use std::io::Write;
+    use self::chrono::{NaiveDateTime, NaiveDate, NaiveTime};
+
+    use sqlite::Sqlite;
+    use sqlite::connection::SqliteValue;
+    use types::{Date, FromSql, IsNull, Time, Timestamp, ToSql, Text};
+
+    #[cfg(not(feature = "postgres"))]
+    use types;
+
+
+    const SQLITE_DATETIME_FMT: &'static str = "%Y-%m-%d %H:%M:%S";
+    const SQLITE_DATE_FMT: &'static str = "%Y-%m-%d";
+    const SQLITE_TIME_FMT: &'static str = "%H:%M:%S";
+
+    #[cfg(not(feature = "postgres"))]
+    expression_impls! {
+        Date -> NaiveDate,
+        Time -> NaiveTime,
+        Timestamp -> NaiveDateTime,
+    }
+
+    #[cfg(not(feature = "postgres"))]
```

Comment:
Can we move these up somewhere outside of the adapter specific modules so there's one clear place it occurs regardless of the backend as long as `#[cfg(feature = "chrono")]` is true?